### PR TITLE
Update run script to start both TW and Publisher containers

### PR DESCRIPTION
**Pull request by ddaina**
Script can be used to start both TaskWorker and Publisher container variants. If not specified otherwise, script will deploy image from `cmssw` docker hub repository:

```
[crab3@crab-dev-tw01 ~]$ ./runContainer.sh -h

Usage: runContainer.sh -v v3.201118 -s TaskWorker
	-v TW/Publisher version
	-s which service should be started: Publisher, Publisher_scheddy, Publisher_asoless, Publisher_rucio or TaskWorker
	-r docker hub repo, if not provided, default is 'cmssw'
```

Usage examples:
**Start TaskWorker:**
```
[crab3@crab-dev-tw01 ~]$ ./runContainer.sh -v v3.201118 -s TaskWorker
526c34919761e4d06cca92f032bb65ec96a9aecfdf25936246193e2a4eac6593
Sleeping for 3 seconds.
Running containers:
CONTAINER ID        IMAGE                            COMMAND                  CREATED             STATUS              PORTS               NAMES
526c34919761        cmssw/crabtaskworker:v3.201118   "/bin/sh -c 'sh star…"   5 seconds ago       Up 3 seconds                            TaskWorker
```
**Start Publisher_schedd:**
```
[crab3@crab-dev-tw01 ~]$ ./runContainer.sh -v v3.201118 -s Publisher_schedd
7c292eee5a8701d353b27d5acc41c266e04a81fab5ca55a07b4c269e449133e9
Sleeping for 3 seconds.
Running containers:
CONTAINER ID        IMAGE                            COMMAND                  CREATED             STATUS              PORTS               NAMES
7c292eee5a87        cmssw/crabtaskworker:v3.201118   "/bin/sh -c 'sh star…"   5 seconds ago       Up 3 seconds                            Publisher_schedd
```

**Start unknown service:**
```
[crab3@crab-dev-tw01 ~]$ ./runContainer.sh -v v3.201118 -s Unknown
Unknown is not a valid service to start. Specify whether you want to start one of the 'Publisher' variants or 'TaskWorker'.

Usage: runContainer.sh -v v3.201118 -s TaskWorker
	-v TW/Publisher version
	-s which service should be started: Publisher, Publisher_scheddy, Publisher_asoless, Publisher_rucio or TaskWorker
	-r docker hub repo, if not provided, default is 'cmssw'
```

**Start service without creating directories first:**
```
[crab3@crab-dev-tw01 ~]$ ./runContainer.sh -v v3.201118 -s Publisher_rucio
Make sure to create needed directories before starting container. Missing directory: /data/container/Publisher_rucio/cfg
```

**Use image from your own Docker Hub repo:**
```
[crab3@crab-dev-tw01 ~]$ ./runContainer.sh -v v3.201117 -s Publisher -r ddaina
Unable to find image 'ddaina/crabtaskworker:v3.201117' locally
v3.201117: Pulling from ddaina/crabtaskworker
Status: Downloaded newer image for ddaina/crabtaskworker:v3.201117
a1d2bc56e8f265cc7d9efa9037c8d72dd818dfeebbdba2774eb50eb4a930475b
Sleeping for 3 seconds.
Running containers:
CONTAINER ID        IMAGE                             COMMAND                  CREATED             STATUS              PORTS               NAMES
a1d2bc56e8f2        ddaina/crabtaskworker:v3.201117   "/bin/sh -c 'sh star…"   8 seconds ago       Up 4 seconds                            Publisher
```

**Trying to run without providing all mandatory variables:**
```
[crab3@crab-dev-tw01 ~]$ ./runContainer.sh -v v3.201118
Make sure to set both -v and -s variables.

Usage: runContainer.sh -v v3.201118 -s TaskWorker
	-v TW/Publisher version
	-s which service should be started: Publisher, Publisher_scheddy, Publisher_asoless, Publisher_rucio or TaskWorker
	-r docker hub repo, if not provided, default is 'cmssw'
```

### CODE REVIEW

**Review by belforte — COMMENTED** [review]
Perfect.

**Review by belforte — APPROVED** [review]

**Comment by belforte** [comment]
Thanks @ddaina You can merge whenever convenient
I am humbled by your bash skills, may I interest you in making the TW/Publ start.sh a bit more friendly ?

File: src/script/Container/runContainer.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  echo -e "\t-v TW/Publisher version"
+  echo -e "\t-s which service should be started: Publisher, Publisher_scheddy, Publisher_asoless, Publisher_rucio or TaskWorker"
+  echo -e "\t-r docker hub repo, if not provided, default points to 'cmssw'"
+  exit 1
+  }
+
+while getopts ":v:s:r:h" opt
+do
+    case "$opt" in
+      h|--help) helpFunction ;;
```
Comment (belforte) [inline src/script/Container/runContainer.sh:16]: I do not think that `|--help` does anything here. If on indicate `--help` or anything of the stort `--string` the script always reports `Unimplemented option: --`

Comment (ddaina) [reply src/script/Container/runContainer.sh]: you are right, thanks.

**Comment by ddaina** [comment]
> may I interest you in making the TW/Publ start.sh a bit more friendly

sure, I'd suggest to define/list things that should be done there in a separate GH issue.

**Pull request merged**